docs/integration/kubernetes.md

```diff
@@ -192,598 +192,593 @@ Container and Kubernetes metrics are collected along with labels and tags, which
 
 ### Storage Metrics
 
-|Name|Type|Unit|Numeric Type|Label|Description|
-|----|----|----|------------|-----|-----------|
-|kubernetes.pvc.available|gauge|bytes|long|available bytes|number of available bytes in the volume|
-|kubernetes.pvc.used|gauge|bytes|long|used bytes|number of used bytes in the volume|
-|kubernetes.pvc.capacity|gauge|bytes|long|volume capacity|the capacity in bytes of the volume|
+|Name|Type|Unit|Numeric Type|Label| Description                           |
```

Comment:
This table has more columns

Reply:
I don't really see that the `Label` column is useful at all, removed the column